tests/components/wemo/__init__.py

```diff
@@ -0,0 +1,78 @@
+"""The tests for WeMo component."""
+
+from homeassistant.components.wemo import WEMO_MODEL_DISPATCH
+from homeassistant.components.wemo.const import DOMAIN
+
+from tests.async_mock import patch
+from tests.common import MockConfigEntry
+
+
+class DeviceTests:
+    """Basic name/id/info tests for all wemo device entities."""
+
+    DEVICE_MODEL = "WeMoDevice"
+    DEVICE_SERIAL_NUMBER = "wemodevice_serialnumber"
+    DEVICE_NAME = "wemodevice_name"
+
+    def test_name(self, wemo_device):
```

Comment:
Please only use standalone pytest test functions, and not class based tests. See eg:
https://github.com/home-assistant/core/blob/fca071742dd9b0827497ced9cb607f4373d0fada/tests/components/manual/test_alarm_control_panel.py#L88-L125